proto/healthchecks/health_checks.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2025 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package org.lfedge.eve.healthchecks;
+option go_package  = "github.com/lf-edge/eve-api/go/healthchecks";
+option java_package = "org.lfedge.eve.healthchecks";
+
+// Request payload for the POST /api/v1/edgeDevice/healthchecks endpoint.
+//
+// ZHealthCheckMsg encapsulates periodic health check reports for critical
+// system hardware components. Each message typically contains status updates
+// for key hardware elements, such as ECC memory and storage devices. These
+// reports are generated at a low frequency (e.g., once every 24 hours) and
+// are not retransmitted in case of network failure, as their data remains
+// relatively stable.
+//
+// The message is secured within a TLS session, which is bound to the device
+// certificate to ensure integrity and authenticity.
+message ZHealthCheckMsg {
+  string devID = 1;
```

Comment:
For these new messages/fields can you use the suggested lower_snake_case (even though we are not changing the use in existing messages)